docs/reference/ml/df-analytics/apis/delete-trained-models-aliases.asciidoc

```diff
@@ -0,0 +1,62 @@
+[role="xpack"]
+[testenv="platinum"]
+[[delete-trained-models-aliases]]
+= Delete Trained Models Aliases API
+[subs="attributes"]
+++++
+<titleabbrev>Delete Trained Models Aliases</titleabbrev>
+++++
+
+Deletes a trained models alias.
```

Comment:
Is it "model's" possessive or singular "Deletes a trained model alias"

Reply:
definitely